taskcat/_amiupdater.py

```diff
@@ -346,120 +341,13 @@ def _determine_templates(self):
-                    account_id=boto3_cache.account_id(region_profile),
-                    partition=boto3_cache.partition(region_profile),
-                    profile=region_profile,
-                    _boto3_cache=boto3_cache,
-                    taskcat_id=taskcat_id,
-                )
-                defined_regions[legacy_optin_region] = region_object
-
-        # Step 3: Anything explicit in auth needs to be added.
-
-        for auth_region, auth_profile in self.config.config.general.auth.items():
-            if auth_region not in defined_regions.keys():
-                region_object = RegionObj(
-                    name=auth_region,
-                    account_id=boto3_cache.account_id(auth_profile),
-                    partition=boto3_cache.partition(auth_profile),
-                    profile=auth_profile,
-                    _boto3_cache=boto3_cache,
-                    taskcat_id=taskcat_id,
-                )
-                defined_regions[auth_region] = region_object
-
-        # FInally, prune anything that's not available via describe-regions or auth.
-
-        prune_these_regions = []
-
-        available_and_auth_regions = list(
-            set(available_regions + list(self.config.config.general.auth.keys()))
-        )
-
-        for _x in defined_regions:
-            if _x not in available_and_auth_regions:
-                prune_these_regions.append(_x)
-
-        for prune_region in prune_these_regions:
-            del defined_regions[prune_region]
-
-        return defined_regions
-
     def _determine_templates_regions(self):
         templates = []
 
         for tc_template in self.template_list:
-            _t = Template(
-                underlying=tc_template, regions_with_creds=list(self.regions.keys())
-            )
+            _t = Template(underlying=tc_template)
             templates.append(_t)
 
-        regions_without_creds = set()
-        regions_excluded = set()
-        for template in templates:
-            for region in template.regions_without_creds:
-                if region in self.EXCLUDED_REGIONS:
-                    regions_excluded.add(region)
-                else:
-                    regions_without_creds.add(region)
-
-        if regions_excluded:
-            LOG.info(
-                "FYI - Your templates use the following regions,"
-                "however no credentials were detected"
-            )
-            LOG.info(", ".join([r.upper() for r in regions_excluded]))
-            LOG.info(
-                "These regions are not within the default AWS Partition. Continuing..."
-            )
-
-        if regions_without_creds:
-            LOG.error(
-                "Your templates use the following regions"
-                "and no credentials were detected for them"
-            )
-            LOG.error(", ".join([r.upper() for r in regions_without_creds]))
-            LOG.error("This can lead to inconsistent results. Not querying the API.")
```

Comment:
I think these messages have value and I'd prefer not to remove them.

Reply:
regions/code pairs that don't get results, because of missing creds.ncorrect filters/no ami avail, get a warning message already. In my use case, where you need multiple profiles in different partitions for things to work, just warning about this and not erroring out is ideal.

Reply:
also, I think I've stripped out all the logic that detects credentials-> regions.